### ISSUE TITLE
SSL certificate expiry dates do not match

### ISSUE THREAD

**Issue by ravenzachary** [Requarks/wiki#2945]
**Describe the bug**
If you look at the certificate expiration date on the Let's Encrypt SSL Certificate in the browser and then compare that the certificate expiration date in the Admin > SSL section of the Wiki, the dates do not match. Currently, the Admin panel is showing an expiration date of 29 January 2021 and the browser cert details lists 22 April 20201. 

**To Reproduce**
1. Load the wiki site with SSL and view the SSL certificate expiration date in the browser.
2. Go to the SSL section of the Admin area of your wiki and view the SSL certificate expiration date on this page.
3. Compare them - they will not be the same.

**Expected behavior**
The SSL certificate expiration date should be the same.

**Screenshots**
![Screen Shot 2021-01-22 at 10 19 23](https://user-images.githubusercontent.com/645975/105529591-735b9480-5c9b-11eb-9e6d-b045bd1a5914.png)

**Host Info (please complete the following information):**
 - OS: Ubuntu 19.09 x64
 - Wiki.js version: 2.5.159
 - Database engine: PostgreSQL 11.5

**Additional context**

**Comment by cashewnuts** [comment]
This problem causes using up the rate limit of Let's encript.
If I switch wiki.js server on and off on daily basis, it will end up the error at 5 days later.

<img width="1210" alt="image" src="https://user-images.githubusercontent.com/44954891/106335399-1598ff00-62d0-11eb-9862-9223c2801ed5.png">

https://letsencrypt.org/docs/rate-limits/

**Comment by PaulKlumpp** [comment]
I experience the same problem. It will try to update the cert far too often.

**Comment by JetForMe** [comment]
Came here to post the same screenshot :)

**Comment by dazzag24** [comment]
Same behavior here.  Just upgraded to wiki.js version 2.5.201 and I still see the issue.
![image](https://user-images.githubusercontent.com/1055831/121004793-ff6fe380-c786-11eb-835c-1e52dc0b965a.png)

**Comment by rogersgt** [comment]
@NGPixel Is there any update on this? My company's wiki site is completely down due to this. Is there at least a workaround for this?

**Comment by NGPixel** [comment]
@rogersgt A simple restart of Wiki.js will trigger a certificate renewal. However it seems you exceeded the API limit so that won't work. I suggest disabling HTTPS for now (see https://docs.requarks.io/troubleshooting#how-to-manually-disable-https-ssl-redirection) and consider using a reverse proxy like Cloudflare or put an nginx instance in front.

**Comment by snarum** [comment]
Doen anyone know when the certificate is actually attempted renewed? to me it looks like the code is either run when the server is started, or when the Renew button is clicked in the gui. But I can't find any code that would run the certificate check regularly.

Right now my SSL page says the certificate is expired yesterday, and letsdebug.org says there is no renewal requests.

**Comment by NGPixel** [comment]
@snarum That's correct. There's no automatic renewal at the moment and yes, it needs to be added.

**Comment by jonystorm** [comment]
Hi all! I have a problem directly related to this one, I'm using versión 2.5.275, and got the email from let's encrypt my current cert only has a few days left, I have the "Renew certificate" button on the SSL admin section but the request triggered from clicking it returns an error of "Empty response", any suggestions? or anyone else having this issue?

**Comment by scott-dunt** [comment]
Suggest you restart your docker containers.. I hit that error before and restarting the containers cleared it up.

**Comment by FiretronP75** [comment]
I confirm this is still a problem. For me it seems to be showing the issued date rather than the expiration date.

**Comment by charlezkwan** [comment]
@NGPixel , any update on automatic renewal certificate feature?

**Comment by miller-coffee** [comment]
I can confirm this is still a problem with Wiki.js 2.5.300

**Comment by integrity-develop** [comment]
I can confirm this issue is still present in Wiki.js 2.5.301. Will this issue cause SSL requests to begin failing when the displayed date is passed? This is the first SSL I have issued for my Wiki.js deployment and I can't quite tell if this is just an issue with what is displayed in the interface or if it is an actual configuration problem that is going to manifest as failed requests. The newly issued SSL certificates display as fine in a browser. Appreciate any information someone can give me on this, and will be investigating nginx implementation in the meantime.

**Comment by marcolussetti** [comment]
This issue is still present years later in 2.5.314

**Comment by TimoKruth** [comment]
Verified on current `main` (`6f042e97`, 2026-08-06): the underlying mismatch is **still present**.

The Admin SSL query in `server/graph/resolvers/system.js` returns `WIKI.config.letsencrypt.payload.expires` when Let's Encrypt is enabled. It does not inspect the certificate currently loaded by the HTTPS server or read the served certificate's `notAfter` value.

As a result, the admin page can continue showing a stored ACME payload expiry that differs from the certificate actually being served, especially after renewal/reload state diverges. A live Let's Encrypt issuance was not performed in this local check, but the reported stale-data code path remains unchanged.
